### PR TITLE
added user show pages and fixed sign in

### PR DESCRIPTION
**Pull request by clb840212**

### CODE REVIEW

**Comment by clb840212** [comment]
coverage passed

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# class 
+class UsersController < ApplicationController
+  def index
+  end
```
Comment (jdmacmurtrie) [inline app/controllers/users_controller.rb]: If this is empty, it should be `def index; end`

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  <li>
+    Show: <%=review.program.title%> <br>
+    Review: <%=review.headline%> <br>
+    Why?: <%=review.body %> <br>
+      Rating: <%=review.rating %> <br>
+      Vote Count: <%=review.vote_count%>
+  </li>
+</ul>
+<%end%>
+<%end%>
```
Comment (jdmacmurtrie) [inline app/views/users/show.html.erb]: Please fix your indentation at the end and your spacing after a bunch of your erb tags: `<% end %>`

File: spec/features/user_edit_spec.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+#   #
+#   #   expect(page).to have_content("Current password can't be blank")
+#   # end
+#     #
+#     # scenario 'user edits profile page' do
+#     #   visit '/users/edit'
+#     #
+#     #   expect(page).to have_content("Edit User")
+#     # end
+# end
```
Comment (jdmacmurtrie) [inline spec/features/user_edit_spec.rb]: Are you using this?  If you're waiting on something else to happen first, leave a note.  Otherwise, eliminate it.

**Pull request merged**